CHANGELOG.adoc

```diff
@@ -53,7 +53,7 @@ The changes are:
 - https://github.com/eclipse-syson/syson/issues/960[#960] [general-view] In the selection dialog of the creation tools for `Stakeholders` and `Actors`, display possible `Part Usage` candidates in a tree instead of a list.
 - https://github.com/eclipse-syson/syson/issues/1012[#1012] [general-view] Allow package nodes to be smaller than their default size
 - https://github.com/eclipse-syson/syson/issues/1006[#1006] [details] Display `FeatureValue.isDefault` and `FeatureValue.isInitial` in Core tab of `FeatureValue` concept.
-
+- https://github.com/eclipse-syson/syson/issues/1007[#1007] Improve direct edit on Feature elements to be able to set the isDefault and isInitial properties
```

Comment:
Could you also update the documentation in `user-manual/hands-on/how-tos/model-management.adoc` please?

Reply:
Could you add '`' around Feature and isDefault  and isInitial  please?